action.yml

```diff
@@ -218,11 +218,24 @@ runs:
             echo "git diff found modified files when it should not have:"
             echo $GIT_DIFF
             echo "sha1sum of files that are different:"
+            COPIED_COUNT=0
             for FILEPATH in $(git diff --cached --name-only); do
               if [[ -f $FILEPATH ]]; then
                 sha1sum $FILEPATH
+                # Only copy if the file is less than 10 megabytes to prevent malicous behaviour
+                # Only copy a max of 10 files, also to prevent malicious behaviour
+                MEGABYTES=$(ls -l --b=M "$DIST_DIR/$FILEPATH" | cut -d " " -f5 | sed "s/M//")
+                if (($MEGABYTES >= 10)); then
+                  echo "File $FILEPATH is larger than 10 megabytes, not copying"
+                elif (($COPIED_COUNT >= 10)); then
+                  echo "More than 10 files have been copied, not copying $FILEPATH"
```

Comment:
If the limit is 10, how would it ever copy more than 10 as the message suggests?

Reply:
The message is slightly off - same with the MB thing - but it's just for debugging an intermittent problem we're having when the dist file built by CI doesn't match what we have locally. The main thing for this message is "too many files", not the actual number.